db/migrate/20151102115238_add_email_to_doctors.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToDoctors < ActiveRecord::Migration
+  def change
+    add_column :doctors, :email, :string
```

Comment:
why do you need this? just add this column to your `20151102105148_create_doctors.rb`